systemd/ignition-quench.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Ignition (record boot completion)
+Documentation=https://github.com/coreos/ignition
+ConditionKernelCommandLine=coreos.ignition-first-boot
```

Comment:
i wonder if we should just make this ConditionFileExists=/boot/.ignition-first-boot

Reply:
^ agreed. No need for grub/02_coreos_firstboot then.

Reply:
Is ConditionFirstBoot not being used anymore?

Reply:
I wanted to explicitly error out if they're somehow out of sync.

Reply:
> when will ConditionFileExists be evaluated?

I think @jlebon recently tested this and verified it was right before the unit executed.

Reply:
Yeah, see https://github.com/dustymabe/ignition-dracut/pull/9#issuecomment-398165136.